srcpkgs/fsearch/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'fsearch'
+pkgname=fsearch
+version=0.1beta2
+revision=1
+build_style=gnu-configure
+hostmakedepends="gettext gettext-devel glib-devel po4a automake autogen autoconf-archive pkg-config intltool glib-devel"
```

Comment:
Duplicated deps, unnecessary ones, please clean this up.

Reply:
`hostmakedepends="autogen automake pkg-config gettext-devel glib-devel autoconf-archive intltool"`
Is ok?

Reply:
@xtraeme Done!